### PR TITLE
use new central for buildkite

### PR DESCRIPTION
**Pull request by juliasloan25**
## Purpose 
Use new central cluster and climacommon modules

Follows template of SurfaceFluxxes.jl [update](https://github.com/CliMA/SurfaceFluxes.jl/pull/162) and ClimaComms.jl [update](https://github.com/CliMA/ClimaComms.jl/pull/69)

### CODE REVIEW

**Review by Sbozzolo — COMMENTED** [review]
Looks good. I think that the job has been waiting because there are no GPU nodes available.

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -15,8 +15,6 @@ env:
   JULIA_NVTX_CALLBACKS: gc
   JULIA_CUDA_MEMORY_POOL: none
```
Comment (Sbozzolo) [inline .buildkite/pipeline.yml:16]: Can you remove this line?

File: .buildkite/longruns/pipeline.yml [hunk 2]
```diff
@@ -18,8 +18,6 @@ env:
   JULIA_NVTX_CALLBACKS: gc
   JULIA_CUDA_MEMORY_POOL: none
```
Comment (Sbozzolo) [inline .buildkite/longruns/pipeline.yml:19]: Can you remove this line?

File: .buildkite/longruns/pipeline.yml [hunk 3]
```diff
@@ -18,8 +18,6 @@ env:
...
   JULIA_MPI_HAS_CUDA: "true"
```
Comment (Sbozzolo) [inline .buildkite/longruns/pipeline.yml:20]: This too.

File: .buildkite/pipeline.yml [hunk 4]
```diff
@@ -15,8 +15,6 @@ env:
...
   JULIA_MPI_HAS_CUDA: "true"
```
Comment (Sbozzolo) [inline .buildkite/pipeline.yml:17]: This too.

File: .buildkite/pipeline.yml [hunk 5]
```diff
@@ -15,8 +15,6 @@ env:
   JULIA_NVTX_CALLBACKS: gc
```
Comment (Sbozzolo) [inline .buildkite/pipeline.yml:15]: This is set twice.

**Comment by juliasloan25** [comment]
@Sbozzolo CI failed with:
> ERROR: Unable to locate a modulefile for 'julia/1.9.3'
>/groups/esm/slurm-buildkite/bin/build_history: line 5: exec: julia: not found

https://buildkite.com/clima/climacoupler-ci/builds/2743#018de7f9-0edd-44be-bd93-7b9322ad960d

**Comment by Sbozzolo** [comment]
I don't know why it's not working. I am retrying building to see if it was a filesystem issue.

**Pull request merged**